lib/Conversion/ImportVerilog/Expressions.cpp

```diff
@@ -428,6 +428,35 @@ struct ExprVisitor {
     return builder.create<moore::ExtractOp>(loc, type, value, lowBit);
   }
 
+  Value visit(const slang::ast::MemberAccessExpression &expr) {
+    llvm::SmallVector<const slang::ast::ValueSymbol *> symbolVec;
+    const slang::ast::Expression *exprIt = &(expr.value());
+    std::string concatName = {};
+    std::string memberName(expr.member.name);
+    concatName = memberName + concatName;
+    symbolVec.push_back(
+        static_cast<const slang::ast::ValueSymbol *>(&expr.member));
+    while (true) {
+      std::string structName((*exprIt).getSymbolReference()->name);
+      concatName = structName + "." + concatName;
+      symbolVec.push_back(static_cast<const slang::ast::ValueSymbol *>(
+          exprIt->getSymbolReference()));
+      if (exprIt->kind == slang::ast::ExpressionKind::MemberAccess) {
+        exprIt = &(exprIt->as<slang::ast::MemberAccessExpression>().value());
```

Comment:
If I remember correctly, `if(auto exprIt = exprIt.as_if<slang::ast::ExpressionKind::MemberAccess>)` could be used to estimate whether the kind is `MemberAccess` and transform.

Reply:
Sorry for that, I consider the `exprIt` as the `Symbol`.

Reply:
That's all right